spec/date.html

```diff
@@ -430,9 +443,14 @@ <h1>Temporal.Date.prototype.difference ( _other_ [ , _options_ ] )</h1>
         1. Let _temporalDate_ be the *this* value.
         1. Perform ? RequireInternalSlot(_temporalDate_, [[InitializedTemporalDate]]).
         1. Perform ? RequireInternalSlot(_other_, [[InitializedTemporalDate]]).
-        1. Let _largestUnit_ be ? ToLargestTemporalUnit(_options_, « *"hours"*, *"minutes"*, *"seconds"*, *"milliseconds"*, *"microseconds"*, *"nanoseconds"* », *"days"*).
-        1. Let _result_ be ? DifferenceDate(_other_.[[ISOYear]], _other_.[[ISOMonth]], _other_.[[ISODay]], _temporalDate_.[[ISOYear]], _temporalDate_.[[ISOMonth]], _temporalDate_.[[ISODay]], _largestUnit_).
-        1. Return ? CreateTemporalDuration(_result_.[[Years]], _result_.[[Months]], _result_.[[Weeks]], _result_.[[Days]], 0, 0, 0, 0, 0, 0).
+        1. Let _calendar_ be _temporalDate_.[[Calendar]].
+        1. Let _calendarID_ be ? CalendarToString(_calendar_).
+        1. Let _otherCalendar_ be _other_.[[Calendar]].
+        1. Let _otherCalendarID_ be ? CalendarToString(_otherCalendar_).
+        1. If _calendarID_ ≠ _otherCalendarID_, throw a *RangeError* exception.
+        <!-- 1. Let _largestUnit_ be ? ToLargestTemporalUnit(_options_, « *"hours"*, *"minutes"*, *"seconds"*, *"milliseconds"*, *"microseconds"*, *"nanoseconds"* », *"days"*). -->
+        1. Return ? CalendarDateDifference(_calendar_, _other_.[[ISOYear]], _other_.[[ISOMonth]], _other_.[[ISODay]], _temporalDate_.[[ISOYear]], _temporalDate_.[[ISOMonth]], _temporalDate_.[[ISODay]], _options_).
+        <!-- 1. Return ? CreateTemporalDuration(_result_.[[Years]], _result_.[[Months]], _result_.[[Weeks]], _result_.[[Days]], 0, 0, 0, 0, 0, 0). -->
```

Comment:
Should this line just be deleted?

Reply:
It should be moved once I specify the ISO8601 calendar.